development/otp-input-app/pages/otp-input/index.mdx

```diff
@@ -95,8 +96,17 @@ This example demonstrating the accepting the only number type values with suppor
 ```js path=../../components/otp-input/ReuseNumberWithPasteExample name=ReuseNumberWithPasteExample
 ```
 
+#### Email Verification
+We have made a email verification component using ReuseOTPInput. We have already made the email state for you. Just give your email state and add the functionality to this component and you can use it in your application.
+<EmailVerification/>
 
-
+```js path=../../components/otp-input/EmailVerification name=EmailVerification
+```
+#### Round and Color Fill
+We made this OTP component to have a beautiful rose color background with red borders on focus which you can customize to whatever color you want in inputClassname prop. Also the OTP container is made to be rounded.
```

Comment:
Pls add a code block here with the inputClassName prop: It will give more idea where and what classnames to add.